components/integrations/bots/add_bot/add_bot.jsx

```diff
@@ -0,0 +1,516 @@
+                    <Tooltip id='removeIcon'>
+                        <FormattedMessage
+                            id='bot.remove_profile_picture'
+                            defaultMessage='Reset Default Icon'
+                        />
+                    </Tooltip>
+                )}
+            >
+                <a
+                    className={'bot-profile__remove'}
+                    onClick={this.setDefault}
+                >
+                    <span>{'×'}</span>
+                </a>
+            </OverlayTrigger>
+        );
+        let imageStyles = null;
+        if (this.props.bot && !this.state.pictureFile) {
+            imageURL = Utils.imageURLForUser(this.props.bot.user_id);
+        } else {
+            imageURL = this.state.image;
+            imageStyles = this.state.orientationStyles;
+            if (this.state.image === BotDefaultIcon) {
+                removeImageIcon = null;
+            }
+        }
+
+        return (
+            <div className='backstage-content'>
+                <BackstageHeader>
+                    <Link to={`/${this.props.team.name}/integrations/bots`}>
+                        <FormattedMessage
+                            id='bots.manage.header'
+                            defaultMessage='Bot Accounts'
+                        />
+                    </Link>
+                    {subtitle}
+                </BackstageHeader>
+                <div className='backstage-form'>
+                    <form
+                        className='form-horizontal'
+                        onSubmit={this.handleSubmit}
+                    >
+                        <div className='form-group'>
+                            <label
+                                className='control-label col-sm-4'
+                                htmlFor='username'
+                            >
+                                <FormattedMessage
+                                    id='bots.add.username'
+                                    defaultMessage='Username'
+                                />
+                            </label>
+                            <div className='col-md-5 col-sm-8'>
+                                <input
+                                    id='username'
+                                    type='text'
+                                    maxLength='64'
+                                    className='form-control'
+                                    value={this.state.username}
+                                    onChange={this.updateUsername}
+                                />
+                                <div className='form__help'>
+                                    <FormattedMessage
+                                        id='bot.add.username.help'
+                                        defaultMessage='Usernames must be all lowercase. They cannot be longer than 21 characters and can only contain letters, numbers, periods, hyphens, and underscores.'
```

Comment:
```suggestion
                                        defaultMessage='You can use lowercase letters, numbers, periods, dahses, and underscores.'
```

Reply:
Also added based on new username help texts, applying here for consistency https://projects.invisionapp.com/share/B7R4BUHYZCS#/screens/353427679